apps/web/src/components/layout/components/v2/HeaderNav.tsx

```diff
@@ -19,18 +25,57 @@ import { WorkflowHeaderBackButton } from './WorkflowHeaderBackButton';
 import { SupportModal } from '../SupportModal';
 
 export function HeaderNav() {
-  const { currentUser } = useAuth();
+  const { currentUser, currentOrganization } = useAuth();
   const [isSupportModalOpened, setIsSupportModalOpened] = useState(false);
   const isV2Enabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_V2_ENABLED);
 
-  useBootIntercom();
```

Comment:
@jainpawan21 I think we should still boot intercom for all of our product marketing activities for the time being.